.gitlab-ci-docker/nginx/site.conf

```
@@ -0,0 +1,9 @@
+server {
+    listen       8000;
```

We could use port 80 internally - the port used by the guest system will be another one anyway.
```suggestion
    listen       80;
```